added code for game of fifteen

rishav-pandey opened the pull request:
please review my code and if you find it good, please merge it to master branch

AdiChat commented on code/unclassified/fifteen/fifteen.c:
```diff
@@ -0,0 +1,291 @@
+ 
+#define _XOPEN_SOURCE 500
+
+#include <cs50.h>
```
Where is `cs50.h`? Additionally, we accept only library which are readily available and are not locally developed. Kindly modify your code to remove this dependency 👍

rishav-pandey replied:
You will find the instruction note for including cs50.h library. This is very simple yet efficient way to use the functions instead of scanf(). As we know scanf() has very adverse effect while using it in complex program.

rishav-pandey commented:
@AdiChat 
Hey, as per your suggestion I'm working on it make to compatible by using only general library.
I have removed the code for game of fifteen temporary.
And also added caesar cipher and vigenere cipher code in c
please have a look on it and also merge it with master branch

AdiChat commented on code/cryptography/ciphers/caesar_cipher/caesar.c:
```diff
@@ -0,0 +1,50 @@
+#include <stdio.h>
+//#include <cs50.h>
```
remove this comment

rishav-pandey replied:
removed the comment :+1:

AdiChat commented on code/cryptography/vigenere_cipher/vigenere_cipher.c:
```diff
@@ -1,59 +1,83 @@
-  char p[100] = "Meet me at 11:00 AM sharp.";
-  
-    if(!isalphastr(k))
+    //check if user give more or less than two arguments.
```
Explicitly defining the key and message makes the code easier to understand. So, changes to this part of code is not necessary 👍

rishav-pandey replied:
this is not my code.
I have completely changed vigenere code and implemented my new one.

AdiChat commented:
Kindly remove the binary file `caesar` and make changes according to comments 👍

rishav-pandey commented:
@AdiChat 
As per your suggestion I have removed the binary file and also removed the comment.

AdiChat commented:
Looks good 👍 

Hope you are enjoying your journey with **OpenGenus**

rishav-pandey commented:
Thanks @AdiChat 
I will give my best in Open source Community and also want to learn more and more things. As I'm new in the field of Computer Science :+1:

AdiChat commented:
Glad to know 😍
